test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,38 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNotNull();
+        
+        // only one null
+        assertHasNull((Object) null);
+        
+        // two null
+        assertHasNull(null, null);
+        
+        // only one not null
+        assertNotNull(1);
+        assertNotNull("");
+        assertNotNull("abc");
+
+        // all objects not null
+        assertNotNull("abc", "ab", "a");
+        assertNotNull(1, 2);
+        assertNotNull("abc", "abc");
+        assertNotNull("abc", "", "abc", "ABC");
+        assertNotNull("", "abc", "a", "abc");
```

Comment:
Quite thorough, however many redundancy.
Does each additional test case have chance of finding a new bug?